src/index.js

```diff
@@ -156,21 +186,21 @@ const processors = {
                 }),
                 _.mapValues(error => {
                     if (_.startsWith('json/', error.ruleId)) return error;
-                    const prefixLength = preprocessorTemplate.indexOf(preprocessorPlaceholder);
 
-                    return _.assign(error, {
+                    const newError = _.assign(error, {
                         column: error.column - (error.line === 1 ? prefixLength : 0),
-                        endColumn: error.endColumn - (error.endLine === 1 ? prefixLength : 0),
-                        fix:
-                            error.fix &&
-                            _.assign(error.fix, {
-                                range: error.fix.range.map(location => location - prefixLength)
-                            })
```

Comment:
This by the way didn't work. For `error.fix=undefined` `_.assign` resulted in a property `fix: undefined`, which isn't compatible with eslint.